www/talp03/sp/Controller/makeOrder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+session_start();
+
+require '../Model/OrderDB.php';
+require '../Model/ProductDB.php';
+require '../Model/UserDB.php';
+
+$priceSum = $_GET['price'];

```

Comment:
jaktoze se mi podarilo objednat za 7 Kc?

![image](https://github.com/nvbach91/4IZ278-2023-2024-LS/assets/20724910/c585ce91-ea7e-492b-b67d-227dce47723a)

Reply:
problém předpokládám byl, že jsem cenu přenášel pomocí GET což bylo bezpečností riziko. Přesunul jsem to do SESSION